src/app/theme/services/theme.service.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from "@angular/core";
+
+@Injectable({
+  providedIn: "root",
+})
+export class ThemeService {
+
+  public static default = 'light';
+
+  public get current(): string {
+    return localStorage.getItem('theme') ?? ThemeService.default;
```

Comment:
I understand that you allow users to manually switch between light/dark mode.
Would be nice if you could respect the browser's appearance first.

Would be great if you could add this.
If not, I can work on this later.